### PR TITLE
[FIX][NSA-353] add delivery execution fields back to cli

### PR DESCRIPTION
**Pull request by andreluizmachado**
Ticket: https://oat-sa.atlassian.net/browse/NSA-353

## What's Changed

The reason which caused this issue is the ticket RE-255 which was supposed to fix an issue on the export UI. The side effect was that the fields were also removed from CLI. Now both bugs should be fixed with these changes

Refs: https://oat-sa.atlassian.net/browse/RE-255
Refs: https://oat-sa.atlassian.net/browse/NSA-353

## How to test
- please check: https://oat-sa.atlassian.net/browse/RE-255
- please check: https://oat-sa.atlassian.net/browse/NSA-353

### CODE REVIEW

File: model/export/SingleDeliveryResultsExporter.php [hunk 1]
```diff
@@ -136,7 +136,8 @@ public function getColumnsToExport(): array
                 $columns = array_merge(
                     $this->columnsProvider->getTestTakerColumns(),
                     $this->columnsProvider->getDeliveryColumns(),
-                    $variables
+                    $variables,
+                    $this->columnsProvider->getDeliveryExecutionColumns()
```
Comment (Karol-Stelmaczonek) [inline model/export/SingleDeliveryResultsExporter.php:140]: Just to be clear, array_merge overwrite array keys with same names so the last one are preserved. Variable $variables (duh!) previously was last one and now it isn't and that could have some repercussions. We should watch out for that but probably that's nothing.

**Review by Karol-Stelmaczonek — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by pnal — CHANGES_REQUESTED** [review]
Please change target branch

File: model/export/SingleDeliveryResultsExporter.php (hunk 1 above)
Comment (pnal) [reply model/export/SingleDeliveryResultsExporter.php:140]: Just left a comment in slack about UI export feature behavior that might be affected by this change

**Review by bugalot — APPROVED** [review]
approved

**Pull request merged**